ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthFeeHistoryTest.java

```diff
@@ -85,10 +86,14 @@ public void params() {
   public void allFieldsPresentForLatestBlock() {
```

Comment:
Reckon it's worth having tests that cover Paris and Shanghai?

Reply:
No, don't think it's worth doing that. Nothing special about the boundaries as far EthFeeHistory is concerned it just passes down the timestamp.